src/test/kotlin/uk/gov/justice/digital/hmpps/incidentreporting/resource/NomisSyncResourceTest.kt

```diff
@@ -748,11 +786,6 @@ class NomisSyncResourceTest : SqsIntegrationTestBase() {
               """
               {
                 "reportReference": "112414666",
-                "type": "SELF_HARM_1",
```

Comment:
the test above is already very thorough with field checks so making this one specific purely to descriptions